src/javascript/binary/static_pages/tnc.js

```diff
@@ -18,9 +18,10 @@ const TermsAndConditions = (() => {
     };
 
     const handleActiveTab = () => {
-        const hash    = window.location.hash || '#legal';
-        const menu    = '.tab-menu-wrap';
-        const content = '.tab-content-wrapper';
+        const hash        = window.location.hash.split('&')[0] || '#legal';
+        const sub_content = window.location.hash.split('&')[1];
```

Comment:
can you store `window.location.hash.split('&')` in a separate variable?

Reply:
yup, sure